go.mod

```diff
@@ -0,0 +1,5 @@
+module magefiles
+
+go 1.20
```

Comment:
This should be 1.19.

Reply:
Used go 1.19

Reply:
I want us to hold publication until we know which version we can use. It would be really weird to be on different mechanisms between the authorizer and the directory binding packages.